docs/sources/next/extensions/explore.md

```diff
@@ -323,6 +275,10 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-tracetest</h4>
         <p>Support for Tracetest test execution and tracing client</p>
     </a>
+    <a href="https://github.com/grafana/xk6-ts" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
Extension owner changed this is why name changed from "szkiba/xk6-ts" to "grafana/xk6-ts"